app/src/main/java/com/github/factotum_sdp/factotum/ui/display/DisplayFragment.kt

```diff
@@ -83,9 +128,9 @@ class DisplayFragment : Fragment() {
                 observeBossFolders()
                 setupBossUI()
             }
-            Role.CLIENT -> {
-                observeClientPhotos()
-                setupClientUI()
+            Role.COURIER -> {
+                observeBossFolders()
+                setupBossUI()
             }
```

Comment:
Maybe a better name or a comment would be great when calling this function. It makes it seem as if the when someone has the courier role, he has access to the bossUI